source/catalog.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+
+<body class="page">
+
+  <header class="main-header">
+    <a class="main-header__logo logo">
+      <img class="logo__image" src="img/logo-image.svg" alt="Логотип Кэт энерджи" width="60" height="50">
+      <img class="logo__text" src="img/logo-text.svg" alt="Кэт энерджи">
+    </a>
+    <nav class="main-nav">
+      <ul class="main-nav__list site-list">
+        <li class="site-list__item">
+          <a class="site-list__navigation" href="index.html">
+            главная
+          </a>
+        </li>
+        <li class="site-list__item">
+          <span class="site-list__navigation site-list__item--active">
+            каталог продукции
+          </span>
+        </li>
+        <li class="site-list__item">
+          <a class="site-list__navigation" href="form.html">
+            подбор программы
+          </a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+
+  <main class="main-conteiner catalog">
+    <h1 class="catalog_title">Каталог продукции</h1>
```

Comment:
```suggestion
    <h1 class="catalog__title">Каталог продукции</h1>
```

Элемент от класса отделяется двойным подчёркиванием

Reply:
[Йоксель](https://yoksel.github.io/html-tree/) тут тоже нашла ошибку 
![image](https://github.com/htmlacademy-adaptive/1919993-cat-energy-1/assets/30808818/ae45556e-5182-4a7c-838e-352ed2cf9b47)